lib/class-wp-theme-json-gutenberg.php

```diff
@@ -88,17 +88,17 @@ class WP_Theme_JSON_Gutenberg {
 			'width'  => null,
 		),
 		'color'      => array(
-			'background'     => null,
-			'corePalette'    => null,
-			'coreGradients'  => null,
```

Comment:
I assume these settings were only recently introduced, which is why we don't need migrations.